centos/8/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:8
```

Comment:
Same as for PR #53: will not this pull the amd64 image?

Reply:
CentOS uses multi-arch images now. When that is executed on an ARM64 host, it will automatically pull arm64v8: https://hub.docker.com/layers/centos/library/centos/centos8/images/sha256-85313b812ad747dd19cf18078795b576cc4ae9cd2ca2ccccd7b5c12722b2effd?context=explore

Reply:
Thanks for the clarification!